doc/docs/reference/upgrading.md

```diff
@@ -38,6 +49,7 @@ you have specified them in for instance, a requirements.txt.
     requests==2.18.4
     django-celery==3.2.2
     celery==3.1.25
+    django-compressor==2.2
```

Comment:
I see what you're doing here, but this will mean that we have several other versions of `/docs/x/y/upgrading/` which don't have this line?

If we're doing it, we need to change the other branches as well surely?